contracts/payloads/IGP34/PayloadIGP34.sol

```diff
@@ -0,0 +1,550 @@
+
+    address public constant USDC_ADDRESS =
+        0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48;
+
+    constructor() {
+        ADDRESS_THIS = address(this);
+    }
+
+    function propose(string memory description) external {
+        require(
+            msg.sender == PROPOSER ||
+                msg.sender == TEAM_MULTISIG ||
+                address(this) == PROPOSER_AVO_MULTISIG ||
+                address(this) == PROPOSER_AVO_MULTISIG_2 ||
+                address(PROPOSER_AVO_MULTISIG_3) == PROPOSER_AVO_MULTISIG_3,
+            "msg.sender-not-allowed"
+        );
+
+        uint256 totalActions = 1;
+        address[] memory targets = new address[](totalActions);
+        uint256[] memory values = new uint256[](totalActions);
+        string[] memory signatures = new string[](totalActions);
+        bytes[] memory calldatas = new bytes[](totalActions);
+
+        // Action 1: call executePayload on timelock contract to execute payload related to Lite & Fluid
+        targets[0] = address(TIMELOCK);
+        values[0] = 0;
+        signatures[0] = "executePayload(address,string,bytes)";
+        calldatas[0] = abi.encode(ADDRESS_THIS, "execute()", abi.encode());
+
+        uint256 proposedId = GOVERNOR.propose(
+            targets,
+            values,
+            signatures,
+            calldatas,
+            description
+        );
+
+        require(proposedId == PROPOSAL_ID, "PROPOSAL_IS_NOT_SAME");
+    }
+
+    function execute() external {
+        require(address(this) == address(TIMELOCK), "not-valid-caller");
+
+        // Action 1: Add new Claim Module and update Default Implementation on Lite
+        action1();
+
+        // Action 2: Update v1.1.0 Vaults config
+        action2();
+
+        //Action 3: Add WBTC vault reward vault on WBTC/USDC and WBTC/USDT
+        action3();
+
+        // Action 4: Transfer WBTC and USDC from Treasury to Team Multisig
+        action4();
+    }
+
+    function verifyProposal() external view {}
+
+    /***********************************|
+    |     Proposal Payload Actions      |
+    |__________________________________*/
+
+    /// @notice Action 1: Add new Claim Module and update Default Implementation on Lite
+    function action1() internal {
+        // set dummy implementation
+        LITE.setDummyImplementation(0x35E7961BE97ccba901Cf14b73202d58f7a33d46d);
+
+        // add Claim Module
+
+        bytes4[] memory sigs_ = new bytes4[](1);
+        sigs_[0] =  bytes4(keccak256("claimFromSpark()"));
+        LITE.addImplementation(0xc10A855055Eb3939FCaA512253Ec3f671C4Ab839, sigs_);
+        
+        address(LITE).call(abi.encode(sigs_[0]));
+    }
+
+    /// @notice Action 2: Update v1.1.0 Vaults config
+    function action2() internal {
+        VaultConfig[] memory configs_ = new VaultConfig[](10);
+
+        uint256 i;
+
+        // ETH_USDC
+        {
+            configs_[i++] = VaultConfig({
+                vault: 0x0C8C77B7FF4c2aF7F6CEBbe67350A490E3DD6cB3,
```

Comment:
@thrilok209 I want to remove manual verification as much as possible so from this proposal can we calculate vault address or fetch from vault factory rather than hardcoding address. For example, here we are updating vaults from 11 to 20.